src/cloud/release-notes/mcd-release-notes.md

```diff
@@ -16,6 +16,19 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## v1.2.1
+*Release date: December 21, 2020*<br/>
+
+-  {:.fix}Fixed PHP-FPM container health check by changing the `WEB_PORT` from `80` to `8080`.<!--MCLOUD-7232-->
+
+-  {:.fix}Fixed an error with invalid volume naming in developer mode.<!--MCLOUD-7442-->
+
+-  {:.new}Added the ability to change the value of the parameters "worker_processes" (option "--nginx-worker-processes" ) and "worker_connections" (option "--nginx-worker-processes" ) for services "tls" and "web". The parameter 'worker_process' retains the ability to set the value "auto".
```

Comment:
What do these options do?  That is, why do you need to add them.  Can you provide link to more information?  Are they options for the `ece-docker build:compos` command?


```suggestion
-  {:.new}Added the ability to change the value of the parameters `worker_processes` (option `--nginx-worker-processes`) and `worker_connections` (option `--nginx-worker-processes` ) for the tls and web services . The parameter 'worker_process' retains the ability to set the value `auto`.
```